deployment/cdk/opensearch-service-migration/dp_pipeline_template.yaml

```diff
@@ -24,9 +28,11 @@ historical-data-migration:
       # Use the same document ID as the source cluster document
       document_id: ${getMetadata("opensearch-document_id")}
       # Example configuration for basic auth
-      username: user
-      password: pass
-      #disable_authentication: true
+      # disable_authentication: true
+      username: "admin"
+      password: "admin"
```

Comment:
Don't we use a different pass for the target cluster?

Reply:
Thanks for catching that! Updated in next commit.